decidim-budgets/config/locales/en.yml

```diff
@@ -142,6 +142,8 @@ en:
         show: See projects
         vote: Vote
         voted_on: You've voted on %{links}
+      last_activity:
+        new_vote_at_html: "<span>New vote at %{link}</span>"
```

Comment:
Can we name it "budgeting vote" in the core? Not to confuse with elections/votings modules.